lib/improver/spotdata/read_input.py

```diff
@@ -184,4 +190,36 @@ def data_from_dictionary(dictionary_data, key):
     if key in dictionary_data.keys():
         return dictionary_data[key]
 
-    raise Exception('Data {} not found in dictionary.'.format(key))
+    raise ValueError('Data {} not found in dictionary.'.format(key))
+
+
+def read_config(file_path):
+    """
+    Read a json file containing configuration information.
+
+    Args:
+    -----
+    file_path : string
+        Path to the configuration file.
+
+    Returns:
+    --------
+    configuration : dict
+        Dictionary containing configuration information.
+
+    Raises:
+    -------
+    IOError : Raised when no file is found at given path.
+
+    """
+    try:
+        config_file = open(file_path, 'r')
+    except:
```

Comment:
Thanks Ed, I've made that change as well and will use the default ValueError that's raised if it fails, rather than my custom one.